schema.sql

```diff
@@ -0,0 +1,105 @@
+DROP DATABASE readme;
+
+CREATE DATABASE IF NOT EXISTS readme
+  DEFAULT CHARACTER SET utf8
+  DEFAULT COLLATE utf8_general_ci;
+
+USE readme;
+
+CREATE TABLE users (
```

Comment:
Рекомендую именовать таблицы в единственном числе
```suggestion
CREATE TABLE user (
```